src/core/account_manager.mjs

```diff
@@ -271,7 +296,7 @@ export class AccountManager {
     let keys
     try {
       keys = await this.getAccountKeys(accountId, nodeClient)
-      this.logger.debug(`retrieved keys for account ${accountId.toString()}`)
+      this.logger.silly(`retrieved keys for account ${accountId.toString()}`)
```

Comment:
Not sure if need this unconventional `silly` log type?

Reply:
yes, I didn' t like the crazy amount of logs these were creating.  I wanted to make them trace, but the npm that winston logger was defaulting to only had silly.

I'll just go ahead and remove them.  They were useful when I was getting it all working, but now that it is pretty solid it can be removed.

Reply:
removed